dev/com.ibm.ws.jdbc/src/com/ibm/ws/jdbc/internal/JDBCDriverService.java

```diff
@@ -1037,6 +1078,189 @@ protected void setSharedLib(Library lib) {
             Tr.debug(this, tc, "setSharedLib", lib);
         sharedLib = lib;
     }
+    
+    @FFDCIgnore(Exception.class)
+    private void setupOracleLogging() {
+        final String method = "setupCustomOracleLogging";
+        
+        final String
+        ORACLELOG_ENABLE_TRACE = "oracle.jdbc.Trace",
+        ORACLELOG_FILE_SIZE_LIMIT = "oracleLogFileSizeLimit",
+        ORACLELOG_FILE_COUNT = "oracleLogFileCount",
+        ORACLELOG_FILENAME = "oracleLogFileName",
+        ORACLELOG_TRACELEVEL = "oracleLogTraceLevel",
+        ORACLELOG_FORMAT = "oracleLogFormat",
+        ORACLELOG_PACKAGENAME = "oracleLogPackageName";
+        
+        Logger parentLogger = Logger.getLogger(ORACLELOG_PARENT_PACKAGENAME);
+        
+        //If the 'oracle' logger isn't using a parent then it's possible the user configured their 
+        //application to send logs somewhere else.  Don't overwrite them.
+        if(!parentLogger.getUseParentHandlers()) {
+            return;
+        }
+        
+        // Fail fast: If some, but not all, required properties were provided tell user and return.
+        if( (System.getProperty(ORACLELOG_FILENAME) != null) == (System.getProperty(ORACLELOG_PACKAGENAME) != null) &&
+            (System.getProperty(ORACLELOG_PACKAGENAME) != null) == Boolean.getBoolean(ORACLELOG_ENABLE_TRACE)) {
+            if(!Boolean.getBoolean(ORACLELOG_ENABLE_TRACE)) { //User enabled logging, but wants logs to go to Liberty
```

Comment:
<NIT>"User enabled logging, but want logs to go to Liberty"  I think you mean "User configured oracle logging, but wants logs to go to Liberty" ? </NIT>